ds701_book/02B-Pandas.qmd

```diff
@@ -4,6 +4,7 @@ jupyter: python3
 code-fold: false
 ---
 
+::: {.content-visible when-profile="web"}
```

Comment:
I'm fine with hiding this in the slide, but it does remind me to point it out to the class to open the lecture with that, so my preference is to leave it on the slides too.

Reply:
Not a problem. I will revise that.